wallet/src/de/schildbach/wallet/data/PaymentIntent.java

```diff
@@ -163,18 +164,16 @@ private Output(final Parcel in) {
     @Nullable
     public final byte[] paymentRequestHash;
 
-    public boolean useInstantX = false;
-
```

Comment:
We no longer need InstantSend indicators for PaymentIntent. This was necessary when it was possible to specify InstantSend as a requirement for a transaction which required an extra fee.